.gitattributes

```diff
@@ -0,0 +1,10 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# to native line endings on checkout.
+*.cs diff=csharp text
```

Comment:
native as in OS native or C# native?
OS native would still be /r/n on windows and /n on linux/osx?